go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	go.uber.org/goleak v1.0.0
 	go.uber.org/multierr v1.4.0
 	go.uber.org/net/metrics v1.3.0
-	go.uber.org/thriftrw v1.25.0
+	go.uber.org/thriftrw v1.27.1-0.20210722163750-b1733d58df3c
```

Comment:
Are we going to have a proper release for this?

Reply:
Yes - this change is pending an official release of ThriftRW.  It exists so that any change on top of this will function correctly and can be viewed without much concern.